feature-tests/utils/ais-types-array.ts

```diff
@@ -0,0 +1,147 @@
+export const ais = {
```

Comment:
```suggestion
export const aisEvents = {
```
This variable could be more descriptive so that it's clearer what the array items are. I think of these as account intervention events so maybe `aisEvents` or `interventionEvents` or something like that?

Reply:
updated